notebooks/introduction/categorical_distribution.ipynb

```diff
@@ -0,0 +1,656 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            for i in range(len(self.thetas)):
</pre><p>You can update self.thetas directly without using a for loop or .at method here. I think you are not using this function anywhere so removing it would be good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/code-first-ml/book1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>